api/src/main/java/org/apache/iceberg/actions/ComputePartitionStats.java

```diff
@@ -34,6 +34,16 @@ public interface ComputePartitionStats
    */
   ComputePartitionStats snapshot(long snapshotId);
 
+  /**
+   * Clears existing partition statistics for all snapshots, allowing the action to recompute them
```

Comment:
Do we want to highlight that this will create more than one commit(one for the removal and one for the update)

Reply:
added